### PR TITLE
Publish containers to Docker Hub

### LINKED ISSUE #32: Re-spin containers
See https://github.com/keycloak-rel-testing/keycloak-rel/issues/5

### LINKED ISSUE #41: Publish container images to Docker Hub

### PR DESCRIPTION
**Pull request by stianst**
Resolves #41, resolves #32

Test run for Operator respin:
https://github.com/keycloak-rel-testing/keycloak-rel/actions/runs/3149406854

Test run for Container respin:
https://github.com/keycloak-rel-testing/keycloak-rel/actions/runs/3149381732

Nightly test run:
https://github.com/keycloak-rel-testing/keycloak-rel/actions/runs/3149446504

Should be merged after https://github.com/keycloak-rel/keycloak-rel/pull/61

### CODE REVIEW

**Comment by stianst** [comment]
Nightly test run:
https://github.com/keycloak-rel-testing/keycloak-rel/actions/runs/3149446504

File: .github/workflows/respin-containers.yml [hunk 1]
```diff
@@ -13,16 +21,75 @@ jobs:
...
+    runs-on: ubuntu-latest
+    needs: [env]
+    outputs:
+      respin: ${{ steps.get-respin.outputs.respin }}
+    steps:
+      - id: get-respin
+        run: |
+          VERSION="${{ inputs.version }}"
+          RESPIN=0
+          VERSIONS=(`curl -L -s 'https://hub.docker.com/v2/repositories/${{ needs.env.outputs.docker-org }}/${{ inputs.repository }}/tags/?page_size=100' | jq -r .results[].name`)
```
Comment (stianst) [inline .github/workflows/respin-containers.yml:34]: Not super happy with tying this to Docker registry APIs, but only sensible non-authenticated API I could find

File: .github/workflows/respin-containers.yml [hunk 2]
```diff
@@ -4,7 +4,15 @@ on:
   workflow_dispatch:
     inputs:
       version:
+        description: 'The version to respin'
```
Comment (jonkoops) [inline .github/workflows/respin-containers.yml:7]:
```suggestion
        description: The version to respin
```

File: .github/workflows/respin-containers.yml [hunk 3]
```diff
@@ -4,7 +4,15 @@ on:
...
         required: true
+      repository:
+        description: 'The container repository to respin'
```
Comment (jonkoops) [inline .github/workflows/respin-containers.yml:10]:
```suggestion
        description: The container repository to respin
```

File: .github/workflows/x-env.yml [hunk 4]
```diff
@@ -15,6 +15,9 @@ on:
       quay-org:
         description: "Quay Organization"
         value: ${{ jobs.env.outputs.quay-org }}
+      docker-org:
+        description: "Docker Organization"
```
Comment (jonkoops) [inline .github/workflows/x-env.yml:19]:
```suggestion
        description: Docker Organization
```

**Review by jonkoops — APPROVED** [review]

**Pull request merged**